x-pack/plugins/lists/server/services/exception_lists/delete_exception_list_items_by_list.ts

```diff
@@ -9,7 +9,7 @@ import type { ListId, NamespaceType } from '@kbn/securitysolution-io-ts-list-typ
 import { getSavedObjectType } from '@kbn/securitysolution-list-utils';
 import { asyncForEach } from '@kbn/std';
 
-import { SavedObjectsClientContract } from '../../../../../../src/core/server/';
+import { SavedObjectsClientContract } from '../../../../../../src/core/server';
```

Comment:
NOTE: These have to be fixed to fit the regular expression we introduce within `moduleNameMapper`